requirements.txt

```diff
@@ -39,6 +39,7 @@ sendgrid==6.9.7
 slowapi==0.1.5
 snowflake-sqlalchemy==1.4.3
 sqlalchemy[asyncio]==1.4.27
+sqlalchemy-citext==1.8.0
```

Comment:
This packages adds a `citext` type that sqlalchemy recognizes.  It's a simple package with 1-2 classes and is MIT licensed.

https://github.com/mahmoudimus/sqlalchemy-citext